### ISSUE TITLE
No field for string body param in Swagger

### ISSUE THREAD

**Issue by dimparf** [scalatra/scalatra#752]
Scalatra 2.5.4 Swagger 2.0
My docs code:
```
val predictWithBody =
    (apiOperation[Prediction]("Get category, sentiment and named entities as prediction JSON")
      summary "Get prediction"
      notes "You must have a API key and authorization token"
      parameter pathParam[String]("id").description("Project id")
      parameter bodyParam[String].description("Text for natural language processing")) //BUG HERE
```
When running my project and show Swagger docs, i do not see field for body (only description for body)

**Comment by takezoe** [comment]
This is my result of your definition on Scalatra 2.5.4:
![2017-11-14 11 00 36](https://user-images.githubusercontent.com/1094760/32759134-2fcf71de-c92b-11e7-9369-e2740814cc1a.png)

**Comment by dimparf** [comment]
@takezoe Yes in generated JSON its ok. But in UI no field for input body text.
This is my screen:
![screenshot_20171115_135429](https://user-images.githubusercontent.com/226784/32818106-06f259e2-ca0d-11e7-860f-1932fd976c4f.png)

**Comment by takezoe** [comment]
Hmm... I'm not sure about Swagger UI. Do you know what JSON is required in this case?

**Comment by dimparf** [comment]
@takezoe Before bodyParam[String] bug was fixed this field was present in UI, after fixing field missing...

**Comment by takezoe** [comment]
Following parameter definition worked fine on Swagger UI. I fixed in #759 and Scalatra 2.6.2 which includes this fix will be release soon.

```javascript
parameters: [
  {
    "name": "body",
    "description": "You must have a API key and authorization token",
    "required": true,
    "in": "body",
    "schema": {
      "type": "string"
    }
  }
]
```
